Source/WebKit/Shared/Cocoa/ArgumentCodersCocoa.mm

```diff
@@ -562,6 +562,17 @@ static void encodeSecureCodingInternal(Encoder& encoder, id <NSObject, NSSecureC
     encoder << (__bridge CFDataRef)[archiver encodedData];
 }
 
+#if ENABLE(DATA_DETECTION) || ENABLE(REVEAL)
+static bool haveSecureActionContext()
```

Comment:
Why not just make this `inline` instead of `static`?  Seems very inline-able.

Reply:
It will be inlined by the optimizer.  "inline" doesn't really do anything in cases like this.